components/eam/src/physics/cam/physpkg.F90

```diff
@@ -1175,6 +1184,8 @@ subroutine phys_run1_adiabatic_or_ideal(ztodt, phys_state, phys_tend,  pbuf2d)
 
     nstep = get_nstep()
     zero  = 0._r8
+    dummy_cflx = 0._r8
+    dummy_landfrac = 0._r8
```

Comment:
`dummy_landfrac` does not appear to be used.

Reply:
this was originally included so that the old `aoa_tracers` module could be used from `FIDEAL`. We don't do this anymore, so I'll just remove the variable.